synapse/lib/view.py

```diff
@@ -367,8 +367,9 @@ async def eval(self, text, opts=None):
 
         self.core._logStormQuery(text, user, opts.get('mode', 'storm'))
 
-        info = {'query': text, 'opts': opts}
-        await self.core.boss.promote('storm', user=user, info=info)
+        taskiden = opts.get('task')
+        taskinfo = {'query': text, 'opts': opts}
```

Comment:
Opts feels erroneous here - it is the only place we have `opts` being plumbed.

Reply:
Agreed. It was already plumbed that way but def feels incorrect. I'll remove, thanks!